integration/test_list.py

```diff
@@ -28,14 +28,15 @@
 
 @inline_callbacks
 @pytest_twisted.inlineCallbacks
-def test_list(request, reactor, temp_dir, introducer_furl, flog_gatherer):
+def test_list(request, reactor, tahoe_venv, temp_dir, introducer_furl, flog_gatherer):
     """
     'magic-folder list' happy-path works
     """
 
     with start_action(action_type=u"integration:test_list:zelda", include_args=[], include_result=False):
         zelda = yield util.MagicFolderEnabledNode.create(
             reactor,
+            tahoe_venv,
             request,
             temp_dir,
             introducer_furl,
```

Comment:
Could this use one of our fixtures, instead of open-coding it?

Reply:
You mean "zelda" or ...? I suspect the point here was to avoid polluting the "alice" / "bob" clients with extra folders (but also, not really relevant to these changes, right?) Or did you mean something specifically about the `tahoe_venv`..?

Reply:
Oh. I guess because those are session scoped? It isn't clear why this needs its own node, but other tests don't.

Reply:
This PR is about Tahoe versions. That's not immediately clear to me either.

I'm sure there are many improvements we can make to the integration-tests (esp. since they're pretty "tahoe"-centric still, having come from there) but we can ignore that here right?